ui/src/pages/Dashboard.vue

```diff
@@ -22,6 +22,13 @@
         </div>
       </template>
     </template>
+    <MarketingBanner title="Ready to scale?" subtitle="Automations, Serverless Work Pools, RBAC, and dedicated support on Prefect Cloud ">
+        <template #actions>
+          <p-button to="https://www.prefect.io/cloud" target="_blank" primary>
+            Upgrade to Cloud
+          </p-button>
+        </template>
+      </MarketingBanner>
```

Comment:
```suggestion
    <MarketingBanner 
      title="Ready to scale?" 
      subtitle="Automations, Serverless Work Pools, RBAC, and dedicated support on Prefect Cloud"
    >
      <template #actions>
        <p-button to="https://www.prefect.io/cloud" target="_blank" primary>
          Upgrade to Cloud
        </p-button>
      </template>
    </MarketingBanner>
```

Just fixing spacing for the linter